expflag/flags.go

```diff
@@ -7,11 +7,13 @@ type Flag string
 const (
 	Example  Flag = "example"
 	UnivKeys Flag = "univ-keys"
+	UikLogs  Flag = "uik-logs"
```

Comment:
nit: https://go.dev/wiki/CodeReviewComments#initialisms
```suggestion
	UIKLogs  Flag = "uik-logs"
```

also, I'm wondering if there's a different way to describe this than "logs" and "logs support" since it doesn't have anything to do with application logs, and the table itself only records the last status/result, and maintains no running "log" of events.

Maybe `uik_requests` for the table, and the expflag I'm less sure, maybe `uik-debug`?